src/snowflake/snowpark/types.py

```diff
@@ -309,6 +309,26 @@ def __init__(self, fields: Optional[List["StructField"]] = None) -> None:
             fields = []
         self.fields = fields
 
+    def add(
```

Comment:
How about calling it `append` since it appends the new item at the end.

Reply:
I had the same thought. I'll rename it and let snow convert team know